bricks.js

```diff
@@ -208,4 +208,75 @@ class SideTube {
             ctx.strokeRect(this.BB.x - this.game.camera.x, this.BB.y, this.BB.width, this.BB.height);
         }
     };
-};
\ No newline at end of file
+};
+
+class FireBar {
+    constructor(game, x, y) {
+        Object.assign(this, { game, x, y });
+        let brick = new Brick(this.game, this.x, this.y, 3, "None");
+        this.game.addEntity(brick);
+
```

Comment:
![image](https://user-images.githubusercontent.com/61642215/108308012-d689fb00-7163-11eb-9bc8-da89d2ccfa93.png)
https://www.mariowiki.com/Fire_Bar#Super_Mario_Bros.
I see that your hard-coding the length/number of fireballs. but I was not sure if its always 6 fireballs and I I THINK that its implying that they are variable so you might need to add a 4th parameter or something for the number of fireballs? I'm honestly not sure tho.

Reply:
I'm pretty sure that every firebar has 6 fireballs in the version of the Super Mario series that we are implementing.

Reply:
![image](https://user-images.githubusercontent.com/62678592/108567319-e832e600-72bc-11eb-823f-7d5282a8e000.png)